### PR TITLE
Introduce Utility Method to Replace Default Table Name

### PR DESCRIPTION
**Pull request by marcusdacoregio**
Closes gh-2256

### CODE REVIEW

**Comment by marcusdacoregio** [comment]
Hey @philwebb, it would be great to get your feedback on this to see how it fits what Spring Boot expects.

File: spring-session-jdbc/src/main/java/org/springframework/session/jdbc/util/JdbcSchemaUtils.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	/**
+	 * Loads the content of the provided schema resource and replaces the
+	 * {@link JdbcIndexedSessionRepository#DEFAULT_TABLE_NAME} by the provided table name.
+	 * @param schemaResource the schema resource
+	 * @param tableName the table name to replace
+	 * @return the schema resource with the table name replaced
+	 */
+	public static Resource replaceDefaultTableName(Resource schemaResource, String tableName) throws IOException {
+		String schemaScript = new String(schemaResource.getInputStream().readAllBytes());
```
Comment (philwebb) [inline spring-session-jdbc/src/main/java/org/springframework/session/jdbc/util/JdbcSchemaUtils.java:44]: We should probably do `try (InputStream inputStream = schemaResource.getInputStream()) {...}` to ensure the stream is closed.

File: spring-session-jdbc/src/main/java/org/springframework/session/jdbc/util/JdbcSchemaUtils.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		String newSchema = schemaScript.replace(JdbcIndexedSessionRepository.DEFAULT_TABLE_NAME, tableName);
+		return new ByteArrayResource(newSchema.getBytes());
```
Comment (philwebb) [inline spring-session-jdbc/src/main/java/org/springframework/session/jdbc/util/JdbcSchemaUtils.java:46]: We might want to read and write with `StandardCharsets.UTF_8`.

**Comment by philwebb** [comment]
@marcusdacoregio It looks good to me, we should be able to make use of it without issue. I've added a couple of minor review comments.

**Pull request merged**